### PR TITLE
safe deployments code lab

### PR DESCRIPTION
**Pull request by tomaslin**

### CODE REVIEW

File: guides/tutorials/codelabs/safe-deployments/index.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+title:  "Safe Deployments"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+In this codelab you will improve a deployment process by adding safeguards to your deployments.
+
+This codelab is a continuation of the Kubernetes: code to prod codelab. It assumes you have an application set up and ready to deploy.
```
Comment (ttomsu) [inline guides/tutorials/codelabs/safe-deployments/index.md]: link to codelab

File: guides/tutorials/codelabs/safe-deployments/index.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+# Setup
```
Comment (ttomsu) [inline guides/tutorials/codelabs/safe-deployments/index.md]: The page title is at H1. I and others have been using H2 for main sections throughout the docs to differentiate.

File: guides/tutorials/codelabs/safe-deployments/index.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+
+You can click on `Skip Execution Window` to start the stage right away.
+
+Go back and modify the execution window so your stage falls within the current window.
+
+Execute your pipeline again, you should see your deployment continue without waiting.
+
+## Adding a Manual Judgment to deployment pipelines
+
+Manual Judgments serve as a gates for your pipeline. You can add manual judment stages to interrupt the pipeline execution to run a manual check. This is great for teams that have a manual or out of bound QA process.
```
Comment (ttomsu) [inline guides/tutorials/codelabs/safe-deployments/index.md]: out of... band?

**Review by ttomsu — APPROVED** [review]

File: guides/tutorials/codelabs/safe-deployments/index.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
+---
+layout: single
+title:  "Safe Deployments"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+In this codelab you will improve a deployment process by adding safeguards to your deployments.
```
Comment (anotherchrisberry) [inline guides/tutorials/codelabs/safe-deployments/index.md]: add a comma: `In this codelab, you will`...

**Pull request merged**